utils/build/virtual_machine/weblogs/nodejs/provision_test-app-nodejs-unsupported-defaults.yml

```diff
@@ -21,8 +21,8 @@ lang_variant:
 weblog:
     name: test-app-nodejs-unsupported-defaults
     #All of these branches are going to install unsupported versions of nodejs
-    exact_os_branches: [ubuntu22_amd64, ubuntu22_arm64, ubuntu21, ubuntu20_arm64, ubuntu20_amd64, centos_7_amd64]
-    install: 
+    exact_os_branches: [ubuntu24_amd64, ubuntu24_arm64, ubuntu22_amd64, ubuntu22_arm64, ubuntu21, ubuntu20_arm64, ubuntu20_amd64, centos_7_amd64]
```

Comment:
It should be "ubuntu24". Check this code: https://github.com/DataDog/system-tests/blob/d203319f9d9cdd250f3f526950a9af37d3f87fbf/utils/_context/virtual_machines.py#L458

Reply:
I'm afraid I don't understand. 
* That line already had entries `ubuntu22_amd64, ubuntu22_arm64` so I now added similar entries for `ubuntu24_amd64, ubuntu24_arm64`.
* I don't want to have this run for all `ubuntu24` branch as `ubuntu24_10amd64` and `ubuntu24_10arm64` are also branch `ubuntu24` and they should run the unsupported-defaults.
Basically, the `exact_os_branches` value here should be the same as `excluded_os_branches` in the other webapp, shouldn't it?